form.html

```diff
@@ -0,0 +1,159 @@
+  <head>
+    <meta charset='UTF-8'/>
+    <meta name='description'>
+    <link rel="stylesheet" href="">
+    <title>Form</title>
+  </head>
+  <body>
+    <div class="wrapper"><!-- main page -->
+      <header class="page-header">
+        <nav class="main-nav">
+          <ul class="main-nav__list">
+            <li class="main-nav__item"><a href="index.html">Home</a></li>
+            <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+            <li class="main-nav__item main-nav__item--active"><a href="form.html">Feedback Form</a></li>
+            <li class="main-nav__item">HTML Academy</li>
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+      <main class="main-content">
+        <section class="preamble">
+          <div class="preamble__inner">
+            <h2 class="preamble__header">Leave your feedback</h2>
+            <p class="preamble__paragraph">Help our hotels to become better! Leave a comment about it, also please leave a feedback about the sights you visited</p>
+          </div>
+        </section>
+
+        <form class="form" action="" method="get">
+
+          <section class="form__section">
+            <h2 class="form__section-header">Introduce yourself:</h2>
+            <div class="form__name">
+              <label class="form__label-text" for="name">Name:</label>
+              <input class="form__input-text" id="name" placeholder="Name" name="name" type='text'>
+            </div>
+            <div class="form__surname">
+              <label class="form__label-text" for="surname">Surname:</label>
+              <input class="form__input-text" id="surname" placeholder="Surname" name="surname" type='text'>
+            </div>
+            <div class="form__middle-name">
+              <label class="form__label-text" for="second-name">Middle name:</label>
+              <input class="form__input-text" id="second-name" placeholder="Middle name" name="second-name" type='text'>
+            </div>
+          </section>
+
+          <section class="form__section">
+            <h2 class="form__section-header">Contact Information:</h2>
+            <div class="form__phone">
+              <label class="form__label-tel" for="phone">Contact number:</label>
+              <input class="form__input-tel" placeholder="Phone number" name="phone" type="tel">
+              <p class="form__paragraph--error">Error, double-check the number</p>
```

Comment:
modifier class?

Reply:
![image](https://user-images.githubusercontent.com/6135781/32416696-419aa448-c245-11e7-9fc6-57d5e5983233.png)

Yes I thought that it will be an error message. 
Should I first write:
class="form__paragraph form__paragraph--error" ?

Reply:
Yes, if it could be like you said `form__paragraph form__paragraph--error` or maybe just an error element `form__error` depending on much are you going to reuse from `form__paragraph` or will you be overriding all the properties with `form__paragraph--error`